core/domain/image_services.py

```diff
@@ -0,0 +1,103 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Service for managing operations on images."""
+
+from core.domain import fs_domain
+from core.platform import models
+import feconf
+
+gae_image_services = models.Registry.import_gae_image_services()
+
+
+def get_image_dimensions(filename, exp_id):
```

Comment:
Where's this called from?

Reply:
Actually this is for the other PR i'm working on. In which we add dimensions to the exploration. (Migration Job). This service is actually common to both of them.

Reply:
Done